public/js/booking_modes.js

```diff
@@ -0,0 +1,150 @@
+// JS to handle different modes of booking a table.
```

Comment:
Is calling this file `boking_models.js` is the right? Does model have anything to do with MVC?